src/components/Details/VmDetails/tests/VmDetails.test.js

```diff
@@ -15,6 +15,8 @@ import { getId } from '../../../../selectors';
 
 const testVmDetails = (vm, otherProps) => <VmDetails {...VmDetailsFixture[0].props} vm={vm} {...otherProps} />;
 
+const flushAllPromises = () => new Promise(resolve => setImmediate(resolve));
```

Comment:
you can reuse https://github.com/kubevirt/web-ui-components/blob/master/src/tests/enzyme.js#L3

Reply:
Nice, thanks :exclamation: :+1: